SSClient.py

```diff
@@ -66,7 +69,7 @@ def conv_shorthand(env: str) -> str:
     return env
 
 
-def match_env_with_id(env: str, id: int):
+def match_env_with_id(env: str, id: Optional[int]):
     """Secret Server expects an 'id' in order to get data from a secret.
        To do this you need to"""
```

Comment:
Nice job adding documentation to the methods! This particular text might be flying over my head — does "To do this you need to" refer to passing the `id` to Secret Server?

Reply:
Secret server stores secrets as records with a unique id. In order to access a specific secret you need to know the id. That's why the list secrets command exists. 

Second sentence is partial and can be deleted

Reply:
Updated